apps/console/src/features/groups/components/edit-group/edit-group-users.tsx

```diff
@@ -344,7 +344,11 @@ export const GroupUsersList: FunctionComponent<GroupUsersListProps> = (props: Gr
                                             handleUnassignedItemCheckboxChange(user)
                                         }
                                         key={ index }
-                                        listItem={ user.userName }
+                                        listItem={
+                                            user?.userName?.split("/")?.length > 1
+                                                ? user?.userName?.split("/")[1]
```

Comment:
null and undefined will be handled in the condition iteself.
```suggestion
                                                ? user.userName.split("/")[1]
```